dev/devicelab/lib/framework/utils.dart

```diff
@@ -184,8 +184,8 @@ Future<Process> startProcess(
   final ProcessInfo processInfo = new ProcessInfo(command, process);
   _runningProcesses.add(processInfo);
 
-  process.exitCode.whenComplete(() {
-    print('\n'); // separate the output of this script from subsequent output to make logs easier to read
+  process.exitCode.then((int exitCode) {
```

Comment:
Not sure if it matters for `exitCode`, but the difference between `whenComplete` and `then` is that `whenComplete` will also trigger if there's an error.